content/en/docs/Writing policies/verify-images/sigstore/_index.md

```diff
@@ -693,6 +693,92 @@ verifyImages:
 ...
 ```
 
+## Ignoring Tlogs and SCT Verification
+
+Cosign 2.0 ( Kyverno version >= 1.11 ) verifies rekor entries for both key-based and identity-based signing. To disable this set `ignoreTlog: true` in kyverno policies:
```

Comment:
Do we need to highlight the Kyverno version? This doc will be available for version 1.11 only.

Reply:
okay, in that case, i am removing kyverno version info